src/styles/index.css

```diff
@@ -0,0 +1,128 @@
+@import './reset.css';
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 200px;
```

Comment:
The `max-width` property is not necessary here if you already set the `width` property in the `.card__content` class. It's better to remove it to avoid confusion.